public/mergify-configuration-openapi.json

```diff
@@ -537,6 +537,14 @@
         "$ref": "#/definitions/BranchName"
       }
     },
+    "ReportModeArray": {
+      "$id": "/definitions/ReportModeArray",
+      "description": "list of report modes",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/ReportModeArray"
```

Comment:
This ref points it this itself so it's a loop.
It should be an enum with the list of possible values.